storage/cloud-client/storage_move_file.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2019 Google Inc. All Rights Reserved.
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
```suggestion
# Copyright 2019 Google LLC. All Rights Reserved.
```

We keep the year, but header checker barfed because of the Inc. not the year 😄